index.js

```diff
@@ -119,6 +117,7 @@ mkdirIfNeeded(GENERATED_DIR)
 // Create version directories
 mkdirIfNeeded(`${GENERATED_DIR}/v1`)
 mkdirIfNeeded(`${GENERATED_DIR}/v2`)
+mkdirIfNeeded(`${GENERATED_DIR}/v3`)
```

Comment:
This probably should be driven from the same data structure if possible. So enumerate the array/object and construct all these directories.